contrib/libhdfs3-cmake/CMakeLists.txt

```diff
@@ -105,6 +105,9 @@ set(SRCS
 
 if (ARCH_AMD64)
     find_program(YASM_PATH NAMES yasm)
+    if (NOT YASM_PATH)
+        message(FATAL_ERROR "Please install the Yasm assembler")
```

Comment:
maybe disable building hdfs in this case after logging the message like "Not using hdfs because ..."?

Reply:
I actually don't like that too much. Nobody reads the log messages of all >50 contribs and there is too much potential for surprises. I also checked for other contribs which do `find_program` and could only find "krb5" (which needs awk) and if the dependency is not found CMake configuration simply fails.

Reply:
Ok